debian/compat

```diff
@@ -1 +0,0 @@
-12
```

Comment:
I didn't realize you could drop the `compat` file after depending on `debhelper-compat`, but indeed that's what the manual says now.